src/components/DiffView.tsx

```diff
@@ -0,0 +1,29 @@
+/// <reference path="DiffView.d.ts"/>
```

Comment:
How about creating a `DiffView` folder?

Reply:
Oh, I thought I did that, but I didn't. I guess I only meant to do that. Thanks!

Reply:
I also renamed the `.tsx` file from `DiffView.tsx` to `index.tsx` to match what we did for addons-frontend. I'm not sure if we want to follow that model, or if we'd rather have the component tsx files named after the components (i.e., `DiffView.tsx`) as I did originally.

Reply:
> How do you use Jest typeahead to find its tests? I assume that currently you can type DiffView and find the test, right?

I didn't check that, as there are so few tests I have just been either running them all, or defaulting to what needs to be run based in my changes. If I type `Diff` it does find and run my test, but I cannot see the typeahead output due to the issue you filed about that.

Reply:
ok, if you rename it to `index.tsx` I'd just want to make sure the same thing happens. It probably does.

Reply:
Sorry, rename *what* to `index.tsx`? If you mean the main component file (not the test file), then that's already done.

Reply:
Oh, I see. Any reason why you didn't rename `DiffView/DiffView.test.tsx` to `DiffView/index.test.tsx`? My concern was that type ahead would still let you type `DiffView` to select that test (it should).

Reply:
I was a bit worried about having so many files called `index.*`. In addons-frontend we named our tests using the component names, but I guess that was because they all lived in the same folder (as opposed to beside each component). I've changed the test to be called `index.test.tsx` and Jest still seems to be able to find it with `p`, so I guess that's a better approach.